### ISSUE TITLE
nvda does not read accented characters when using the synthesizer voices windows one core

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#8860]
### Steps to reproduce:
Change the language of windows to Portuguese Brazil including the voices of windows one core.
Change the language of nvda to Portuguese Brazil.
I do not know if assemblies are required for other languages the configuration above reflects the one used on my machine.

I change the voice synthesizer to windows one core, I'm using microsoft voice microsoft Maria or microsoft Daniel microsoft.
Maybe this will work with other languages that use accentuation in the letters.
Now type accented letters in a notepad for example:
á é í ú
â ê
à
ã

Now read these 1 to 1 characters with right and left arrows.
### Actual behavior:
nvda announces only the letters but does not read the accentuation of them
### Expected behavior:
The nvda should read the letters and accents applied to each letter.
If you use espeak in Portuguese, the accent is usually read.
This seems to me to be a problem with the windows voice synthesizer and not with nvda.
But it seems to me that Microsoft can not fix this.
And still these voices of windows one core are common for being of quality and they are gratuitous, and still be like standard in the nvda for windows 10.
So this is affecting a lot of people I believe.
So, although it does not seem like a nvda problem, however, if nvda can do something to fix it, it would be of huge value.
#### NVDA Installed/portable/running from source:
instal
#### NVDA version:
alfa 16164
#### Windows version:
10 17763.55
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
no

**Comment by PratikP1** [comment]
Could you try to reproduce this problem with Narrator and let us know if OneCore voices still exhibit this behavior? If they do, I will follow up with Microsoft. As my command of Portuguese is not good enough, I do not believe I can reproduce this myself. Thank you.

**Comment by fernando-jose-silva** [comment]
Thank you very much.
I did some tests, and with the narrator the problem also exists.

**Comment by PratikP1** [comment]
I have submitted the following feedback. https://aka.ms/AA2yybw

This issue should be closed as Microsoft Most probably needs to fix OneCore behavior.
